### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by minosx31**

### CODE REVIEW

File: docs/team/minosx31.md [hunk 1]
```diff
@@ -9,26 +9,26 @@ Army Information Management System is a desktop address book application built t
...
+  - Managed releases `v1.2.1` and `v1.3.0` on Github
 
 - **Enhancements to existing features**:
-  - *to be added soon*
+  - Allow resizing of the result display pane
 
 - **Documentation**:
   - User Guide:
-    - *to be added soon*
+    - Added documentation for the `copy` feature
```
Comment (niekis) [inline docs/team/minosx31.md:26]: can link the PR to this, as well as the developer guide one

**Review by niekis — CHANGES_REQUESTED** [review]

File: docs/team/minosx31.md [hunk 2]
```diff
@@ -1,34 +1,42 @@
 ---
-layout: page
-title: Neo Ming Hao's Project Portfolio Page
+layout: page title: Neo Ming Hao's Project Portfolio Page
```
Comment (rexcyrio) [inline docs/team/minosx31.md:2]: Not sure if the lack of the line break here will mess up GitHub Pages + Jekyll -- just take note ya?

**Review by rexcyrio — APPROVED** [review]
LGTM :+1:

**Review by rexcyrio — APPROVED** [review]
LGTM :+1:

**Pull request merged**